app/lib/forms/choose_your_npq.rb

```diff
@@ -2,18 +2,42 @@ module Forms
   class ChooseYourNpq < Base
     include Helpers::Institution
 
-    attr_accessor :course_id
+    QUESTION_NAME = :choose_your_npq
 
-    validates :course_id, presence: true
+    attr_accessor QUESTION_NAME
+
+    validates QUESTION_NAME, presence: true
     validate :validate_course_exists
 
     def self.permitted_params
-      %i[
-        course_id
+      [QUESTION_NAME]
+    end
+
+    def question
+      Forms::QuestionTypes::RadioButtonGroup.new(
+        name: :choose_your_npq,
+        options:,
+        opts: { fieldset: { legend: { size: "m", tag: "h1" } } },
+      )
+    end
+
+    def options
+      [
+        build_option_struct(value: "leading_behaviour_and_culture", link_errors: true),
+        build_option_struct(value: "leading_literacy"),
+        build_option_struct(value: "leading_teaching"),
+        build_option_struct(value: "leading_teacher_development"),
+        build_option_struct(value: "senior_leadership"),
+        build_option_struct(value: "headship"),
+        build_option_struct(value: "executive_leadership"),
+        build_option_struct(value: "early_years_leadership"),
+        build_option_struct(value: "early_headship_coaching_offer", divider: true),
```

Comment:
It still feels weird to have the move to a system where we have the keys for courses stored in two locations. Why can't we use a map of courses? Like we previously had, or a map or something on the course model. 

As it stands we now have DISPLAY_NAME_MAPPING and here defining the available courses and the display column on the Course model is now ignored.

Reply:
Well i changed it away from the ID mapping to the keys and unless that key is in the DB there is no way of linking up. 
Think im not understanding what you mean but your suggesting change it back to have id's in the values which then gets the content to be displayed in the `en.yml`.  Which would allow you to look up in the DB as used through most the tests and finding the course etc..

The code has keys for finding the data which arent added in the DB. I just felt linking id's in the code and having them map to courses in the db was a bit of a code smell. Ideally is more readable if the key means something and this is how its found in the db.